### DISCUSSION TITLE
Code to Cloud Updates

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#32194] [Actions]
Hello everyone! and welcome to another Code to Cloud check in. We had a really fruitful month of August with an upbeat live Copilot AMA on Discussions where our product managers answered all of your questions. Missed it ? Check it out [here](https://github.com/orgs/community/discussions/29932). Awesome! I hope you make this September a month to remember. 😄

## New, Larger, Faster and Secure - [Large GitHub Hosted Runners are here](https://github.blog/2022-09-01-github-actions-introducing-the-new-larger-github-hosted-runners-beta/) 🤩

We’re excited to announce the public beta of our larger and powerful [GitHub-hosted runners](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners) for Linux and Windows. Wondering what’s in it for you? 🤔 

### Fixed IP ranges 🌐 
Yes! You heard it right. You’re now empowered to setup a fixed(static) IP range for yourself just by ticking a checkbox ☑️.  Not just that, when enabled, instances of the larger runner will receive an address from a range that is unique to the runner, allowing you to use this range to configure a firewall allow list.

![Screenshot 2022-09-07 at 1 18 29 PM](https://user-images.githubusercontent.com/95834130/188820714-5cb345bb-ba48-4b3a-8f39-9542e36872dc.png)

### Larger machine sizes 🔷 
You’ll now be able to make use of machine sizes up to 64 cores on demand to run workflows, billed by the job minute. ⌨️

![Screenshot 2022-09-07 at 1 19 13 PM](https://user-images.githubusercontent.com/95834130/188820934-d45816c2-f9ee-45ea-8ca6-dc45f635bbe0.png)

### Smarter Admin Control 👀 
As an Admin, you get to decide who has access to larger machine sizes and at what concurrency, providing guard rails on spending. 

To learn more about the larger runner per job minute pricing, check out the [updated pricing docs](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions)
To learn more about using the new larger runners, check out the [overview document](https://docs.github.com/en/actions/using-github-hosted-runners/using-larger-runners).

## [Project ReadME](https://github.com/readme) - [Keep separate codebases in sync with GitHub Actions](https://github.com/readme/guides/fullstory-github-actions) 📬

![Screenshot 2022-09-07 at 1 27 06 PM](https://user-images.githubusercontent.com/95834130/188822853-3b775966-4e09-4ff8-b5f1-4122457300c9.png)
_Artwork: [Ariel Davis](https://arielrdavis.com/)_

Each time I bring you the gems of the open source community bi-weekly, whose contributions move the world forward every day. This week it’s time for a story from Sabrina Li (Software Engineer, FullStory) sharing her experience of implementing [GitHub Actions](https://github.com/features/actions) to boost developer productivity by automating manual tasks. Give it a [read](https://github.com/readme/guides/fullstory-github-actions) and thank me later. 🔖

## From the world of Actions! 💯 

Get alerted when new issues get opened in your repository with this handy [GitHub Action](https://github.com/mxarc/issuetron-3000). This Action triggers a MQTT alert when someone opens an issue in one of your repositories.

## More power 💪 to you! - [Permissions and Org level enhancements for npm registry](https://github.blog/changelog/2022-08-31-packages-fine-grained-permissions-and-organization-level-publishing-are-now-available-for-the-github-packages-npm-registry/)

Fine-grained permissions and organization-level publishing are now available for the GitHub Packages npm registry running it on a new architecture, unlocking great new capabilities. Read more about [working with the GitHub npm registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry).

## Your Feedback Matters! 📍
We are continuously building better resources and documentation for you. Do you have any feedback around docs that needs any improvement or doesn’t cover everything you’d like to see around a specific feature? Feel free to share any and our Docs team would love to review them. Alternatively, you can always [contribute](https://github.com/github/docs/edit/main/content/index.md) directly to any doc by submitting a PR, learn more about [how to contribute](https://github.com/github/docs/blob/main/CONTRIBUTING.md) to docs.

To see what’s next for Actions, visit our [public roadmap](https://github.com/github/roadmap/projects/1?card_filter_query=label%3Aactions). Till Next Time! ☮️ ➡️🚀

**Comment by Anonix666** [comment]
TOLBVSC

**Comment by mohamadkarimpor** [comment]
سلام تراز والت من هک کردن چکار باید کرد من و راهنمایی کنید خواهش می کنم

**Comment by Kai391** [comment]
Hello there,
I am Kai a web developer, how can I access codespaces or code to cloud functionality for my github?

**Reply by RONIE70** [reply]
Personalizing GitHub Codespaces for your account
(https://docs.github.com/codespaces/customizing-your-codespace/personalizing-github-codespaces-for-your-account)
[About personalizing Codespaces](https://docs.github.com/es/codespaces/customizing-your-codespace/personalizing-github-codespaces-for-your-account#about-personalizing-codespaces)
[Settings Sync](https://docs.github.com/es/codespaces/customizing-your-codespace/personalizing-github-codespaces-for-your-account)
[Dotfiles](https://docs.github.com/es/codespaces/customizing-your-codespace/personalizing-github-codespaces-for-your-account)
[Further reading](https://docs.github.com/es/codespaces/customizing-your-codespace/personalizing-github-codespaces-for-your-account)
You can personalize GitHub Codespaces by using a dotfiles repository on GitHub or by using Settings Sync.

GitHub Codespaces is available for organizations using GitHub Team or GitHub Enterprise Cloud. GitHub Codespaces is also available as a limited beta release for individual users on GitHub Free and GitHub Pro plans. For more information, see "[GitHub's products](https://docs.github.com/en/get-started/learning-about-github/githubs-products)."

**Reply by Kai391** [reply]
Thankyou for reply me.
I have my personal GitHub account, so how could I be the member of beta release of codespaces?

**Comment by Gborw** [comment]
Thankyou for reply me.
I have my personal GitHub account, so how could I be the member of beta release of codespaces?

**Reply by RONIE70** [reply]
El acceso a la beta de Codespaces es por invitación, y se puede solicitar [en la página web de GitHub sobre Codespaces aquí](https://github.com/features/codespaces).

Una vez cursada la petición y al cabo de un tiempo, recibiremos un email confirmándonos que hemos sido aceptados y formamos parte de Codespaces.
A partir de aquí, podremos acceder a Codespaces desde GitHub.

Si vamos a un proyecto que tengamos dentro de GitHub, observaremos que en el apartado Code, aparece una nueva opción Open with Codespaces.

**Reply by RONIE70** [reply]
Access to the Codespaces beta is by invitation, and can be requested on the Codespaces GitHub webpage here.

Once the request has been submitted and after some time, we will receive an email confirming that we have been accepted and are part of Codespaces.
From here, we can access Codespaces from GitHub.

If we go to a project that we have within GitHub, we will observe that in the Code section, a new Open with Codespaces option appears.
